### ISSUE TITLE
Android 14 December 2023 OTA crush modem with this module

### ISSUE THREAD

**Issue by syxqpb** [Displax/modem-fix#10]
I install December 2023 features drop OTA on my Pixel 7. When I turn on this extension and reboot phone, my radio signal completely disappeared. On other hand when i turn off extension and reboot phone again internet and calling works as usually.

Phone: Pixel 7
Build number: UQ1A.231205.015
Custom kernel: false
Root Access: true
Mobile provider: A1(CIS)
Module version: v1.9

**Comment by menacius** [comment]
I have exactly the same issue. I've tried to reinstall the mod, but installation failed.

[magisk_install_log_2023-12-16T16.12.30.log](https://github.com/Displax/modem-fix/files/13695868/magisk_install_log_2023-12-16T16.12.30.log)

**Comment by vlad196** [comment]
It's because CFG in module and firmware is different.
Need to add new modified cfg.db from new firmware: https://github.com/Displax/modem-fix/tree/main/cfgs/pixel_7

**Comment by syxqpb** [comment]
> It's because CFG in module and firmware is different. Need to add new modified cfg.db from new firmware: https://github.com/Displax/modem-fix/tree/main/cfgs/pixel_7

Sounds good, but I don't have knowledge how to extract it from firmware by myself

**Comment by vlad196** [comment]
> > It's because CFG in module and firmware is different. Need to add new modified cfg.db from new firmware: https://github.com/Displax/modem-fix/tree/main/cfgs/pixel_7
> 
> Sounds good, but I don't have knowledge how to extract it from firmware by myself

I am currently looking at 4pda myself as the author changes cfg.db. 
So far, I can say that this file is located in `/vendor/firmware/carrierconfig/cfg.db`, but that it. I don't know what to change there.
We have to wait for the author to update. From what I've read, it shouldn't be difficult (I suppose)

**Comment by vlad196** [comment]
Check out the new **fork** of this repository from me.
I'm not quite sure if I've done everything, because I haven't fully figured out the issue, but I've tried and it seems to work for me. Be careful, if you don't want to take risks, then wait for the author

**Comment by azuli** [comment]
any news? 
i changed sim carrier and their system can see that i have a pixel 7 which is not supported and they won't enable 5g on my device..

**Comment by vlad196** [comment]
> any news? i changed sim carrier and their system can see that i have a pixel 7 which is not supported and they won't enable 5g on my device..

no. Now, i use another modem-fix with auto implements changes in new modem firmware

**Comment by denisnzrvs** [comment]
Any alternative fixes for 6a?

**Comment by azuli** [comment]
> > any news? i changed sim carrier and their system can see that i have a pixel 7 which is not supported and they won't enable 5g on my device..
> 
> no. Now, i use another modem-fix with auto implements changes in new modem firmware

does this do the same thing as the module from displax? can you share?

**Comment by vlad196** [comment]
@denisnzrvs @azuli 
https://github.com/andrewz1/modem-patch
There is no description on github, basically he wrote everything on the thematic forum. 
As he claims, his module does not require constant updating, and itself rewrites the necessary lines. I've been using it for 2 months, everything works. 
At one's own risk

**Comment by Merlin-Legend** [comment]
> > any news? i changed sim carrier and their system can see that i have a pixel 7 which is not supported and they won't enable 5g on my device..
> 
> no. Now, i use another modem-fix with auto implements changes in new modem firmware

Hello, Which one you use modem? Can you share it?
